content/installation/upgrade.md

```diff
@@ -51,9 +61,30 @@ Follow the steps below to perform an in-place upgrade:
 2. Run the install script with the appropriate node-selector, tolerations, and `KUBELET_DIR_PATH` environment variables needed for your environment.
 
    ```sh {.copy}
-   curl -sfL https://github.com/minio/directpv/raw/master/docs/tools/install.sh | sh - apply
+   curl -sfL https://github.com/minio/directpv/raw/master/docs/tools/install.sh | sh -s - apply
    ```
 
+#### Retain PodSecurityPolicy support
+
+If you upgrade to 4.1.x and wish to retain support for `PodSecurityPolicy`, complete the following steps.
```

Comment:
maybe?
```suggestion
If you upgrade to 4.1.x and wish to retain support for `PodSecurityPolicy`, complete the following steps:
```

But also, the bold pseudo-headings make this section feel a bit odd. All the headings look alike, I might have expected them to be H3 with two H4s. (Itself not great, yeah.)

Maybe make it an ordered list? Change "Before starting the upgrade" to something like "Back up the existing YAML" ? One of many possible ideas.

Reply:
Reworked these headings. I could make them an h3 and turn the pseudo headings I had into an h4.